code/edgeconsent/build.gradle

```diff
@@ -221,8 +226,8 @@ configurations.all {
 }
 
 dependencies {
-    // TODO: Update the value of mavenCoreVersion in gradle.properties to 2.0.0 when Core 2.0 is published to Maven
     implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
+    implementation "com.adobe.marketing.mobile:edge:${rootProject.mavenEdgeVersion}"
     implementation 'androidx.annotation:annotation:1.5.0'
```

Comment:
The annotation lib version here is different from the other projects, which are using either 1.0.0 or 1.3.0. It shouldn't matter, but wondering if we are standardizing on a specific version of the lib.

Reply:
agh.. missed this one. we'll use min 1.0.0